main.py

```diff
@@ -35,8 +37,33 @@ def convert_website_to_stamp(_website, maximum_pages):
     _extractor = extractor.Extractor(_website.url)
     _website.set_contents(_extractor.extract_html())
 
+    _classifier_and_summarizer = ClassifierAndSummarizer(
+        _website.contents,
+        maximum_pages
+    )
+
+    _classifier_and_summarizer_response \
+        = _classifier_and_summarizer.get_stampified_content()
+
+    if not _classifier_and_summarizer_response["is_stampifiable"]:
+        LOGGER.debug("Website is not stampifiable")
+        return
+
     LOGGER.debug(_website.convert_to_dict())
 
+    _generator = StampGenerator(
+            _website,
+            _classifier_and_summarizer_response["stamp_pages"]
+        )
+    generated_stamp = _generator.generate_stamp()
+
+    try:
+        f = open('stamp_generation/final_generated_stamp.html', 'w')
```

Comment:
Will it be possible to name the generated stamp page according to the name of the input webpage (eg. something like "<name of webpage being stampified>_stamp.html")?

Reply:
Yes, It can be named that way. We can use title of the webpage in place of **_** in **_stamp.html**.

Reply:
Updated accordingly!